components/modal/nz-modal.component.html

```diff
@@ -40,7 +40,7 @@
 
 <!-- [Predefined] Default Modal Content -->
 <ng-template #tplContentDefault>
-  <div *ngIf="nzTitle" class="ant-modal-header">
+  <div #modalHeader *ngIf="nzTitle" class="ant-modal-header" (mousedown)="initDrag($event)" [ngStyle]="{'cursor': cursorStyle}">
```

Comment:
Be care the `modalHeader` may be not exist when `nzTitle` is empty, this can cause side effect errors